messaging/src/main/java/org/axonframework/eventhandling/EventProcessorBuilder.java

```diff
@@ -68,10 +68,24 @@ public EventProcessorBuilder name(@Nonnull String name) {
      * @param eventHandlerInvoker the {@link EventHandlerInvoker} which will handle all the individual
      *                            {@link EventMessage}s
      * @return the current Builder instance, for fluent interfacing
+     * @deprecated in favor of {@link #eventHandlingComponent(EventHandlingComponent)}
      */
+    @Deprecated(since = "5.0.0", forRemoval = true)
```

Comment:
Will be removed in next PRs as well as with the LegacyEventHandlingComponent and EventHandlerInvoker, now I keep it and introduce legacy adapters to limit the scope.